documentation/staging/content/managing-domains/accessing-the-domain/access-admin-console-em.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Access the Administration Console"
+date: 2019-02-23T17:39:15-05:00
+draft: false
+weight: 1
+description: "Access the WebLogic Admin Console to manage domains running in Kubernetes."
+---
+
+You can use the WebLogic Server Administration Console to monitor and manage a WebLogic domain running in Kubernetes.
+
+{{% notice note %}} Do not use the WebLogic Server Administration Console to start or stop servers, or for scaling clusters. See [Starting and stopping servers]({{< relref "/managing-domains/domain-lifecycle/startup#starting-and-stopping-servers" >}}) and [Scaling]({{< relref "/managing-domains/domain-lifecycle/scaling.md" >}}). In addition, if your domain home type is either [Domain in Image]({{< relref "/samples/domains/domain-home-in-image/_index.md" >}}) or [Model in Image]({{< relref "/samples/domains/model-in-image/_index.md" >}}), then do not use the Administration Console to make changes to the WebLogic domain configuration as these changes are ephemeral and will be lost when servers restart. See [Choose a domain home source type]({{< relref "/managing-domains/choosing-a-model/_index.md" >}}).
+{{% /notice %}}
+
+To setup WebLogic Server Administration Console access to a domain running in Kubernetes, you can:
+   * Deploy a load balancer with [ingress path routing rules for non-SSL port](#configure-ingress-path-routing-rules) and [SSL port](#access-the-weblogic-server-administration-console-through-the-ssl-port).
+
+   * Use an [Administration Server `NodePort`](#use-an-administration-server-nodeport).
+
+   * [Use a `kubectl port-forward` connection](#use-a-kubectl-port-forward-connection).
+
+{{% notice warning %}}
+Externally exposing administrative, RMI, or T3 capable WebLogic channels
+using a Kubernetes `NodePort`, load balancer,
+port forwarding, or a similar method can create an insecure configuration.
+For more information, see [External network access security]({{<relref "/security/domain-security/weblogic-channels.md">}}).
+{{% /notice %}}
+
+
+#### Configure ingress path routing rules
+
+1. Configure an ingress path routing rule. For information about ingresses, see the [Ingress]({{< relref "/managing-domains/accessing-the-domain/ingress/_index.md">}}) documentation.
+
+   For an example, see the following `path-routing` YAML file for a Traefik load balancer. If you have multiple domains managed by a single ingress controller, then see the [Host-based routing](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/kubernetes/samples/charts/traefik/README.md#host-based-routing) example or consider using the [Remote Console]({{< relref "/managing-domains/accessing-the-domain/admin-console/_index.md">}}).
+ 
```

Comment:
There seems to be no mention of the need for the 'host based' approach to update DNS, neither here nor in the referenced host based sample. Please strongly consider it. You might just want to include a note in the sample, but IMO, it'd help many folks to also mention it here.  I have seen multiple folks get stuck when they hit this step, and waste substantial amounts of their own time, when they might as well have immediately tried using a different approach instead if they had known about the limitation ahead of time...

Reply:
I have updated the `host-based` routing sample to mention the DNS requirement in [d0e1e48](https://github.com/oracle/weblogic-kubernetes-operator/pull/3525/commits/d0e1e48232ae98af28ae5a02747e3cec4620d13a) and also mentioned it in the note.